google/cloud/spanner_dbapi/cursor.py

```diff
@@ -298,6 +321,11 @@ def fetchone(self):
             return None
 
     def fetchall(self):
+        """Fetch all elements.
```

Comment:
Better use the [PEP-249](https://www.python.org/dev/peps/pep-0249/#fetchall) definition.